mmv1/third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -2685,6 +2737,16 @@ func testAccCheckComputeInstanceHasNetworkIP(instance *compute.Instance, network
 	}
 }
 
+func testAccCheckComputeInstanceHasNetworkPerformanceConfig(instance *computeBeta.Instance, bandwidthTier string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if instance.NetworkPerformanceConfig.TotalEgressBandwidthTier != bandwidthTier {
+			return fmt.Errorf("Wrong network_performance_config.total_egress_bandwidth_tier found: expected %v, got %v", bandwidthTier, instance.NetworkPerformanceConfig.TotalEgressBandwidthTier)
+		}
+
+		return nil
+	}
+}
+
```

Comment:
I assume not gating these behind the beta-only conditional is okay since they aren't a travelled code path in the GA provider.

Reply:
I think we still want to gate everything related to the field just for clarity. It makes the git history a bit cleaner and can provide some clarity for those browsing code in the GA provider itself.

Reply:
This includes the other test function, instance helper functions, but not the HTML documentation. Documentation ends up getting merged for both versions.

Reply:
Got it - I'll take care of that now.